deny.toml

```diff
@@ -90,6 +86,7 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
+unused-allowed-license = "allow"
```

Comment:
This will allow us to not have to remove from the list of approved licenses even it if isn't in use.